### DISCUSSION TITLE
how to implement Unicode symbols for operators

### DISCUSSION THREAD

**Discussion by bend-n** [tonsky/FiraCode#1517] [Q&A]
The readme says: 

> all programming languages should be designed with full-fledged Unicode symbols for operators

Im trying to add these symbols to a font im working on, but theres a few I cant find unicode equivalents for.
I took a look through the font, and noticed that the character for `≥` is implemented, yet theres a ligature for `>=`.
So, should I implement the ligatures, *and* the unicode symbols for my font?
And if so, what are the symbols that are in the unicode standard?

**Comment by tonsky** [comment]
>  So, should I implement the ligatures, and the unicode symbols for my font?

I mean, if you want to. I don’t think many currently existing languages actually use Unicode operators

> And if so, what are the symbols that are in the unicode standard?

There are so many. No way you’ll be implementing all of them. Just pick your subset that you find useful

**Reply by bend-n** [reply]
Yeah, I dont plan to implement all of them, but what are the symbols? I cant find the symbols for === for example, i only can find ⩶.

**Reply by tonsky** [reply]
Not everything has a direct Unicode equivalent

**Reply by bend-n** [reply]
Ok, but the ones that do, what are they?

**Reply by tonsky** [reply]
https://unicode-table.com/
